roofit/roofitcore/src/RooAbsPdf.cxx

```diff
@@ -1738,6 +1743,7 @@ RooFitResult* RooAbsPdf::fitTo(RooAbsData& data, const RooLinkedList& cmdList)
   cfg.nWorkers = pc.getInt("nWorkers");
   cfg.parallelGradient = pc.getInt("parallelGradient");
   cfg.parallelLikelihood = pc.getInt("parallelLikelihood");
+  cfg.logTimings = pc.getInt("logTimings");
```

Comment:
As we discussed on Mattermost, your new feature only works for a `RooSimultaneous`.

It would be better to make that more explicit, sooner than later, so it doesn't cause more confusion.

What about checking if `this` is a RooSimultaneous, and if not you are ignoring `logTimings` and print a warning with `coutW(Minimization)`, telling that LogTimings is not supported for other PDFs? That would be more graceful than writing out empty JSON files/histograms.